Gem/Code/Source/Components/UI/UiSettingsComponent.cpp

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Atom/RHI/Factory.h>
+#include <AzCore/Console/IConsole.h>
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/Serialization/EditContext.h>
+#include <IAudioSystem.h>
+#include <LyShine/Bus/UiButtonBus.h>
+#include <LyShine/Bus/UiElementBus.h>
+#include <LyShine/Bus/UiTextBus.h>
+#include <Source/Components/UI/UiSettingsComponent.h>
+
+namespace MultiplayerSample
+{
+    void MpsSettings::Reflect(AZ::ReflectContext* context)
+    {
+        if (AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serializeContext->Class<MpsSettings>()
+                ->Version(0)
+                ->Field("GraphicsApi", &MpsSettings::m_atomApiType)
+                ->Field("MasterVolume", &MpsSettings::m_masterVolume)
+                ->Field("TextureQuality", &MpsSettings::m_streamingImageMipBias)
```

Comment:
can we add a setting for turning off "Pain" sounds? hehe, or maybe that's not a thing anymore :)

Reply:
Oh, no, I can't add that. That setting wouldn't make sense for most people, we only play the pain sound on your pc...
```
if (getenv("LOGNAME") == "AMZN-alexpete")
{
    TriggerSound("PAIN!!!!");
}
```